scanners/trivy/parser/parser.js

```diff
@@ -2,22 +2,38 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-async function parse(scanResults) {
+async function parse(fileContent) {
   // The first scan always contains the image id a similar format to: "bkimminich/juice-shop:v10.2.0 (alpine 3.11.5)"
-  const imageScanResult = scanResults;
 
-  if (typeof(imageScanResult) === "string") // empty file
-    return [];
+  let scanResults = fileContent;
+  if (typeof fileContent === "string") {
+    if (fileContent.includes("{") && fileContent.includes("}")) {
+      scanResults = JSON.parse(fileContent)
+    } else {
+      // empty file
+      return [];
+    }
+  }
+
+  if (scanResults.hasOwnProperty('ClusterName')) {
```

Comment:
```suggestion
  if (Object.prototype.hasOwnProperty.call(scanResults , 'ClusterName')) {
```
I would avoid calling hasOwnProperty() like so. See https://stackoverflow.com/questions/39282873/object-hasownproperty-yields-the-eslint-no-prototype-builtins-error-how-to

Reply:
Recommendation adopted